lib/http/2/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module HTTP2
-  VERSION = "1.0.2"
+  VERSION = "1.1.0"
```

Comment:
I think version gets bumped automatically with the publisher, doesn't it? I thought I read something about version.rb being assumed.

Reply:
it doesn't (and I'm counting on it, as changes in this file trigger the release workflow).

Reply:
https://github.com/rubygems/release-gem/blob/v1/action.yml

Reply:
https://bundler.io/guides/creating_gem.html#releasing-the-gem

This is what I was thinking of. I guess it's gem bump that could do this. So that can be in the CI if you wanted.

Reply:
Not sure if that's the most feasible. You need a way to tell CI that you want to bump the version, and whether it's a patch/minor/major bump. Bumping locally and triggering the release flow on changes to version.rb seems more straightforward.